home/tests/test_content_import_export.py

```diff
@@ -1185,6 +1185,21 @@ def test_multiple_field_errors(self, csv_impexp: ImportExport) -> None:
             "Validation error: whatsapp_template_category - Select a valid choice. Marketing is not one of the available choices.",
         ]
 
+    def test_import_ordered_content_sets_error(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a broken CSV for ordered content sets shoud throw an exception
+        """
+        csv_impexp.import_file("contentpage_required_fields.csv")
+
+        with pytest.raises(ImportException) as e:
+            content = csv_impexp.read_bytes("ordered_content_broken.csv")
+            csv_impexp.import_ordered_sets(content)
+
+        assert e.value.row_num == 0
+        assert e.value.message == [
+            "Row Test Set has 2 times, 2 units, 3 before_or_afters, 3 page_slugs and 3 contact_fields and they should all be equal."
+        ]
+
     def test_import_ordered_sets_csv(self, csv_impexp: ImportExport) -> None:
         """
         Importing a CSV file with ordered content sets should not break
```

Comment:
This seems like a pretty low bar, and the test appears to do more than that. Can we update the docstring to reflect that?